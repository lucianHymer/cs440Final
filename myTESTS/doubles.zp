//simple.cmm 
//Input: None
//Output: 2017 

double a;
int b;

int calc_c(){
  return a + b;
}

int calc_3(double c){
  return a + b + c;
}


int main ()
{
  int c;
  int d;
  int e;

  a = 2;
  b = 3;

  c = calc_c();
  d = calc_3(4);

  putn (c);
  return 0;
}
