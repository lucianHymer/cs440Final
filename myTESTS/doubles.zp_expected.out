     5 
 9.000 
